component-samples/device/src/main/java/org/fidoalliance/fdo/sample/FdoSysDeviceModule.java

```diff
@@ -137,9 +139,18 @@ public void send(ServiceInfoModuleState state, ServiceInfoSendFunction sendFunct
 
   }
 
+  private String getAppData() {
+    DeviceConfig config = Config.getConfig(RootConfig.class).getRoot();
+    File file = new File(config.getCredentialFile());
```

Comment:
Will `config.getCredentialFile()` return `credential.bin` path or path to 'app-data' folder ?